src/run/handlers/cache.cts

```diff
@@ -476,7 +449,7 @@ export class NetlifyCacheHandler implements CacheHandlerForMultipleVersions {
     await Promise.all(
       tags.map(async (tag) => {
         try {
-          await this.blobStore.setJSON(await this.encodeBlobKey(tag), data)
+          await this.cacheStore.set(tag, data, 'tagManifest.set')
```

Comment:
Just noting that this will add tracing (I believe that's a good thing)

Reply:
Yes, I did want to use this opportunity to capture all blobs operations for debugging purposes